lib/ansible/modules/cloud/amazon/ec2_metric_alarm.py

```diff
@@ -184,7 +184,11 @@ def create_metric_alarm(connection, module):
     insufficient_data_actions = module.params.get('insufficient_data_actions')
     ok_actions = module.params.get('ok_actions')
 
-    alarms = connection.describe_alarms(alarm_names=[name])
+    alarms = None
+    try:
+        alarms = connection.describe_alarms(alarm_names=[name])
+    except BotoServerError as e:
+        module.fail_json(msg=str(e))
```

Comment:
I'd also add the traceback for more detailed info here as well as below.
e.g. something like:
```suggestion
        module.fail_json(msg="Failed to describe alarm %s: %s" % (name, str(e)), exception=traceback.format_exc())
```

Reply:
ok.